### PR TITLE
Add Ada to the list of supported languages (#10475)

### LINKED ISSUE #10475: Add Ada to supported languages for debugging
### Feature Request

Hello everyone!

I suggest to add Ada to the list of supported languages for debugging. Ada is debuggable through GDB, and modifying the extension's `package.json` file to add Ada makes everything work out of the box.

This would be very useful for us since we are developing our own VS Code extension for Ada (https://github.com/AdaCore/ada_language_server): the idea would be to bind `vscode-cpptools` as an `extensionPack` to allow users to easily debug their Ada code, with a very minimal JSON configuration.

### PR DESCRIPTION
**Pull request by AnthonyLeonardoGracio**
To allow debugging Ada projects with GDB. Fixes https://github.com/microsoft/vscode-cpptools/issues/10475

### CODE REVIEW

File: Extension/package.json [hunk 1]
```diff
@@ -3146,6 +3146,7 @@
         "type": "cppdbg",
         "label": "C++ (GDB/LLDB)",
         "languages": [
+          "ada",
```
Comment (sean-mcmanus) [inline Extension/package.json:3149]: Can you move this below "rust"? Unless the order is supposed to be alphabetical?

**Review by sean-mcmanus — CHANGES_REQUESTED** [review]

**Review by WardenGnaw — APPROVED** [review]

**Comment by sean-mcmanus** [comment]
@WardenGnaw Should we add Fortran too? I noticed https://github.com/microsoft/vscode-cpptools/issues/10406 .

File: Extension/package.json (hunk 1 above)
Comment (AnthonyLeonardoGracio) [reply Extension/package.json:3149]: I think it's alphabetical yes. It probably does not change the extension's behavior in any case.

**Comment by WardenGnaw** [comment]
> @WardenGnaw Should we add Fortran too? I noticed #10406 .

I would only add it if we have a request. GDB supports the following languages C, C ++, D, Go, Objective-C,  Fortran, Pascal, Rust, Assembly, Modula-2, and Ada.

I'm hesitant on adding everything unless there is a request for it.

**Review by sean-mcmanus — APPROVED** [review]

**Pull request merged**